javaparser-testing/src/test/java/com/github/javaparser/bdd/steps/ExistenceOfParentNodeVerifier.java

```diff
@@ -461,6 +503,42 @@ public void visit(TypeExpr n, Void arg) {
         }
 
         @Override
+        public void visit(ArrayBracketPair n, Void arg) {
```

Comment:
I assumed this visitor should check all parents, so when I noticed a bunch of visits were missing I added them all.